tests/test_resources/test_clusters/test_cluster.py

```diff
@@ -590,7 +590,16 @@ def status_cli_test_logic(self, cluster, status_cli_command: str):
         assert "node: " in status_output_string
         assert status_output_string.count("node: ") >= 1
 
-        # if it is a GPU cluster, check GPU print as well
+        cloud_properties = cluster.config().get("launched_properties", None)
+        if cloud_properties:
+            properties_to_check = ["cloud", "instance_type", "region", "cost_per_hour"]
+            for p in properties_to_check:
+                property_value = (
```

Comment:
what about the `cloud` property necessitates this?

Reply:
We could add a validation for the `launched_properties` in the status output, but i think it is more of a den test.
The point of test is to make sure that the cloud info is printed as expected to `stdout`. 
Or did you mean something else?

Reply:
i meant what makes cloud differnt from the other properties, but seems like it's just bc of how Sky formats it?

Reply:
yes, i think so. We could also double check it with Caroline.

Reply:
you added this line `cloud = cloud_properties.get("cloud").upper()` in `_print_cloud_properties`. I think this might be in response to me adding the `cloud = str(launched_resource.cloud).lower()` in setting the launched properties cloud attribute?

In the sky status resource, they have the cloud capitalized (e.g. Kubernetes, AWS, GCP, Lambda, etc) but I thought it simpler to just use the lowercase form (since that's what we prefer/use in our examples anyways). I think we can leave it that way for printing out in status too? But if we want the original form then we should update the launched_properties attribute to that too.